pkg/management/component.go

```diff
@@ -391,7 +391,15 @@ func (c *component) genericCreateUser(ctx context.Context, accessToken string, c
 	user api.UserRepresentation, generateUsername bool, generateNameID bool, termsOfUse bool, useOnboardingCheckForExistingUser bool) (string, error) {
 	var userRep = api.ConvertToKCUser(user)
 	userRep.SetAttributeString(constants.AttrbSource, source)
-	userRep.SetAttributeString(constants.AttrbOnboardingStatus, managementOnboardingStatus)
+
+	realmAdminConfig, err := c.configDBModule.GetAdminConfiguration(ctx, targetRealmName)
+	if err != nil {
+		c.logger.Info(ctx, "err", err.Error())
```

Comment:
If there is an error, we should return.

Reply:
Warn seems to me more appropriate to log this error.
Maybe also add an explicit message, that we cannot retireve realmAdmin configuration  for example.